clients/privacy-center/common/hooks/useCookie.ts

```diff
@@ -0,0 +1,49 @@
+import { getCookie, setCookie, Types } from "typescript-cookie";
```

Comment:
neat, this lib is probably a lot better for setting cookies! I remember not using it before since we were only reading, but it's a good call to bring it in now that we are setting as well

Reply:
This is used already in `fides-consent.js` so seemed like a safe bet to use here too

Reply:
FWIW: since the `fides-consent` package already exports `getConsentCookie` and `setConsentCookie` methods, I find this pretty duplicative. That means that we'll be saving two cookies to the browser (`fides_consent` and `fides_user_device_id`) and we'll need to support that going forward too...

The way we've built `fides-consent` right now, though, makes it pretty hard to work on it. This is *dramatically* better on the monorepo branch so within a few weeks this'll be a non-issue, but the timing of this is really unfortunate...

I'd say this: let's get it working, tested, and ready to go... and I _might_ try to refactor this so that we just save this data into the existing cookie. It's *much* more maintainable for us to just extend that format (we're going to need to do that, anyways) instead of trying to maintain two cookie formats